docs/rules/no-empty-component-block.md

```diff
@@ -62,7 +64,15 @@ p {
 
 ## :wrench: Options
 
-Nothing.
+```json
+{
+  "vue/no-duplicate-attributes": ["error", {
+    "autofix": false,
+  }]
+}
+```
+
+- `"autofix"` ... If `true`, enable autofix. (Default: `false`)
```

Comment:
Why does this have to be an option? I think always adding the autofix would be fine and would reduce the complexity a bit.

Reply:
I treated it as providing an auto-fix feature, and upgrading the package wouldn't change the current code. Of course, I think it's also reasonable to directly fix it in the next minor version.

Reply:
I think it's okay if there are no options.
My only concern is that if we have autofix on save enabled in our editor, it will be removed if we temporarily save while writing code.
But other than that, I think it's better to have autofix enabled all the time. If it gets removed, you can probably undo it with Ctrl(Option)+Z.

Reply:
I'll update the doc and tests later